pontoon/base/templates/project_selector.html

```diff
@@ -26,7 +26,7 @@
         {% endif %}
           <span class="name"
             data-slug="{{ p.slug }}"
-            data-details="{{ p.details }}">{{ p.name }}</span>
+            {% if p.details %}data-details="{{ p.details }}"{% endif %}>{{ p.name }}</span>
```

Comment:
Besides translate.html, we also include project_selector.html in locale.html, projects.html and admin.html templates that don't use translate_old.js. In translate.html, p.details are always available.